dataservice/api/common/enumerations.yaml

```diff
@@ -0,0 +1,165 @@
+## ------------ NOTES ----------- ##
+# 'Unknown' - contributor specified this in source data
+# 'Not Reported' - should be used as the default if no value is specified in
+# the column or if this column doesn't exist in source data
+
+## --------- Questions --------- ##
+# How should values be cased - lower case, proper/title case?
+# What should default value be if Not Reported in source data?
+
+study:
+    release_status:
+        choices:
+            - Pending # Not sure about the choices here
+        default: Pending
+
+study_file:
+    availability:
+        choices:
+            - Immediate Download
+            - Cold Storage
+        default: Immediate Download
+
+participant:
+    gender:
+        choices:
+            - Male
+            - Female
+            - Unknown
+            - Not Reported
+        default: Not Reported
+    race:
+        choices:
+          - White
+          - American Indian or Alaska Native
+          - Black or African American
+          - Asian
+          - Native Hawaiian or Other Pacific Islander
+          - Other
+          - Unknown
+          - Not Reported
+        default: Not Reported
+    ethnicity:
+        choices:
+            - Hispanic or Latino
+            - Not Hispanic or Latino
+            - Unknown
+            - Not Reported
+        default: Not Reported
+
+diagnosis:
+    diagnosis_category:
+        choices:
+            - Structural Birth Defect
+            - Cancer
+            - Other
+            - Unknown
+        default: Other
+
+phenotype:
+    observed:
+        choices:
+            - Positive
+            - Negative
+            - Unknown
+        default: Unknown
+
+outcome:
+    vital_status:
+        choices:
+            - Alive
+            - Dead
+            - Unknown
+            - Not Reported
+        default: Not Reported
+    disease_related:
+        choices:
+            - Yes
+            - No
+            - Unknown
+            - Not Reported
+        default: Not Reported
+
+biospecimen:
+    analyte_type:
+        choices:
+            - DNA
+            - RNA
+            - Unknown
+            - Not Reported
+        default: Not Reported
+
+genomic_file:
+    data_type:
+        choices:
+            - Submitted Aligned Reads
+            - Submitted Aligned Reads Index
+            - Simple Nucleotide Variation
+            - Other
```

Comment:
I'm not sure how much value really comes from these choices and they can be hard to define.